### DISCUSSION TITLE
flycam lost connection

### DISCUSSION THREAD

**Discussion by nnup-qert** [time4tea/gopro-dashboard-overlay#159] [Q&A]
My phantom 3s drone is connected to the controller but the light on the controller stays red but on the drone the light is blue and I can control the camera up down and start the motor but when I wifi connection and when I go to the dji go app it says the remote control signal is weak, on the phone screen the image and information of the aircraft cannot be connected, it only shows the controller settings. please help me

**Comment by time4tea** [comment]
Hi - I think you have the wrong place. This isn't a place to discuss GoPro hardware - please contact GoPro support. Hope you find a resolution. Cheers!
